tests/containers/forms/LogActivityForm.test.jsx

```diff
@@ -65,9 +65,10 @@ describe('<LogActivityForm />', () => {
   it('should show the <TextArea/> component when it has loaded', () => {
     expect(wrapper.find('TextArea').length).toEqual(1);
   });
-
-  it('should show the <Buttons/> component when it has loaded', () => {
-    expect(wrapper.find('Buttons').length).toEqual(2);
+  it('should show the <Button/> component when it has loaded', () => {
+    wrapper.setState({ loading: false });
+    const inputComponent = wrapper.find('Button');
+    expect(inputComponent.length).toEqual(2);
```

Comment:
We could also evaluate the html text within the buttons to make sure it aligns with the actual buttons that we expect.

Reply:
I have added a test file for the button component to evaluate the contents of the button. I figured it is better to handle test cases for button content in a separate test file